app/api/login/route.ts

```diff
@@ -0,0 +1,21 @@
+import { OAuth2Client } from "google-auth-library";
+import { NextRequest, NextResponse } from "next/server";
+
+export async function POST(request: NextRequest) {
+    const client = new OAuth2Client();
+    const body = await request.json();
+    const clientId = process.env.GOOGLE_LOGIN_CLIENT_ID;
+    const authResponse = await client.verifyIdToken({
+        idToken: body.token,
+        audience: clientId
+    });
+    const payload = authResponse.getPayload();
+    const userObj = {
+        email: payload?.email,
+        name: {
+            firstName: payload?.given_name,
+            lastName: payload?.family_name
+        }
+    }
```

Comment:
All my same comments still apply [from this comment](https://github.com/jjm3x3/AmazingRaceFantasy/pull/279#discussion_r2062779738)

Reply:
Yeah, when we decide what data we store in our DB and what data should be returned and used on the FE, this will change